packages/scoped-elements/package.json

```diff
@@ -45,6 +46,10 @@
     "@open-wc/dedupe-mixin": "^1.3.0",
     "@webcomponents/scoped-custom-element-registry": "0.0.1"
   },
+  "devDependencies": {
+    "patch-package": "^6.4.7",
```

Comment:
shouldn't this be a prod dependency?

Reply:
I don't think so.... anyway, I'm going to close the PR as is not going to be needed anymore because the change has been approved on the polyfill and merged today.